src/stdatamodels/properties.py

```diff
@@ -571,6 +595,18 @@ def put_value(path, value, tree):
 def merge_tree(a, b):
     """
     Merge elements from tree `b` into tree `a`.
+
+    Parameters
+    ----------
+    a : JSON object tree
+        The tree to merge into.
+    b : JSON object tree
+        The tree to merge from.
+
+    Returns
+    -------
+    a : JSON object tree
+        The merged tree.
```

Comment:
```suggestion
    a : dict
        The tree to merge into.
    b : dict
        The tree to merge from.

    Returns
    -------
    a : dict
        The merged tree.
```
Aren't these just dictionaries?

Reply:
At least while running `src/stdatamodels/jwst/datamodels/tests/test_wcs.py` they are of type `asdf.tags.core.AsdfObject`.  I have updated this docstring to contain that exact type

Reply:
Doesn't the function support any dictionary (and `AsdfObject` acts like a dictionary)? As far as I can tell there's nothing special concerning `AsdfObject` in this function.

Reply:
I am fine with `dict` if you're happier with it, will change